src/Microsoft.SqlTools.Migration/MigrationService.cs

```diff
@@ -599,6 +602,33 @@ public void InitializeService(IProtocolEndpoint serviceHost)
             }
         }
 
+        /// <summary>
+        /// Handle request generate the ARM template.
+        /// </summary>
+        internal async Task HandleGetArmTemplateRequest(
+    string skuRecommendationReportFilePath,
+    RequestContext<string> requestContext)
+        {
+            try
+            {
+                ProvisioningScriptServiceProvider provider = new ProvisioningScriptServiceProvider();
+                List<SkuRecommendationResult> recommendations = ExtractSkuRecommendationReportAction.ExtractSkuRecommendationsFromReport(skuRecommendationReportFilePath);
+                SqlArmTemplate template = provider.GenerateProvisioningScript(recommendations);
+
+                string jsonOutput = JsonConvert.SerializeObject(
+                template,
+                Formatting.Indented,
+                new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore }
```

Comment:
Use Culture = CultureInfo.Invarient/neutral. otherwise double number might get read inappropriately.

Reply:
fixed in another PR